gap/attr.gi

```diff
@@ -1487,3 +1483,50 @@ function(gr)
   od;
   return fail;
 end);
+
+InstallMethod(MaximalAntiSymmetricSubdigraph, "for a digraph",
+[IsDigraph],
+function(D)
+  local n, m, out, i, j;
+
+  n := DigraphNrVertices(D);
+  if n <= 1
+      or (HasIsAntisymmetricDigraph(D) and IsAntisymmetricDigraph(D)) then
+    return D;
```

Comment:
If `n <= 1` or the digraph is already antisymmetric, it's possible that there are still multiple edges, which the documentation forbids!

Reply:
Ooops, I swapped the clauses around here.